galleries.py

```
@@ -463,16 +490,19 @@ def _label(target, page, pages):
             links[-1].append((filename, _label(target, pages, pages),
                                         _title(target, pages, pages)))
             product_ids[filename] = ids
+        
+    _gallery(fileroot, product_ids, catalog, links)
+    
+class FixIndent(yaml.Dumper):
 
-    _gallery(fileroot, product_ids, catalog, links,
-             separators=('|',''), linebreaks=0)
+    def increase_indent(self, flow=False, indentless=False):
```

Ruff (0.14.14)

[error] Boolean default positional argument in function definition

(FBT002)